edit-post/index.js

```diff
@@ -84,16 +83,25 @@ export function initializeEditor( id, post, settings ) {
 		);
 	}
 
-	render(
-		<Editor settings={ migratedSettings || settings } onError={ reboot } post={ post } />,
-		target
-	);
+	return new Promise( ( resolve ) => {
+		domReady( () => {
```

Comment:
Should it be up to the editor to decide the specific time it should be initialized? Or should the prerequisite of the initialization be considered by that which is initializing it (thinking outside the specific "WP + Plugins" context).

Reply:
Good question. you mean moving this to `client-assets` instead. Sounds like a good idea.

Reply:
Now that you asked it this way, it sounds reasonable 😃
I fully focused on fixing the issue with blocks loaded too late and I didn’t think about it.
We will have to make dom-ready its own module but it would happen sooner or later anyway.